frontend/sw360-portlet/src/main/webapp/html/projects/includes/attachmentUsagesRows.jspf

```diff
@@ -60,6 +65,11 @@
                     <sw360:out value="${releaseLink.longName}" maxChar="60"/>
                 </a>
             </td>
+            <td>
+            	<sw360:DisplayEnum value="${releaseLink.releaseRelationship}"/>
+            </td>
+            <td>
+            </td>
```

Comment:
Could you please remove the extra if they are not significant here?

Reply:
removed the additional table data tag.